website/docs/integrations/jira.md

```
@@ -0,0 +1,108 @@
+- Jira Cloud or Jira Data Center with admin access to configure webhooks
+
+---
+
+## Setup
+
+### 1. Get the Webhook URL
+
+In Aurora, go to **Connectors** > **Jira** > connect your account. Once connected, the **Incoming Webhook** card shows your webhook URL. Copy it.
+
+The URL format is:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)